app/main.py

```diff
@@ -1,70 +1,93 @@
+from constants import SCREEN_WIDTH, SCREEN_HEIGHT, DEFAULT_OBJECT_SIZE,\
+    PLAYER_FRONT, PLAYER_BACK, PLAYER_LEFT, PLAYER_RIGHT
+from app.bomb import Bomb
+from app.walls import Wall
 import pygame
+import time
+
 
 from pygame.locals import (
     K_UP,
     K_DOWN,
     K_LEFT,
     K_RIGHT,
     K_ESCAPE,
+    K_SPACE,
     KEYDOWN,
     QUIT,
 )
 
 pygame.init()
 clock = pygame.time.Clock()
-
-
-SCREEN_WIDTH = 650
-SCREEN_HEIGHT = 650
-DEFAULT_OBJECT_SIZE = 50
-
 screen = pygame.display.set_mode([SCREEN_WIDTH, SCREEN_HEIGHT])
 
 
 class Player(pygame.sprite.Sprite):
+    timeout = 0
+
     def __init__(self):
         super(Player, self).__init__()
-        self.surf = pygame.Surface((20, 40))
-        self.surf.fill((0, 255, 0))
+        self.surf = pygame.image.load(PLAYER_FRONT)
         self.rect = self.surf.get_rect()
 
-    def update(self):
-        pass
+    def update(self, pressed_keys):
+        if time.time() - Player.timeout >= 1 and pressed_keys[K_SPACE]:
+            self.plant_bomb()
 
+        if pressed_keys[K_UP]:
+            self.surf = pygame.image.load(PLAYER_BACK)
+            self.rect.move_ip(0, -5)
+            if pygame.sprite.spritecollideany(player, walls):
```

Comment:
You don't check a collision with bombs for now.
Check how it works in the repository for day two )